packages/edit-site/src/components/header/undo-redo/redo.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { redo as redoIcon } from '@wordpress/icons';
+import { displayShortcut } from '@wordpress/keycodes';
+
+export default function UndoButton() {
+	const hasRedo = useSelect( ( select ) => select( 'core' ).hasUndo() );
```

Comment:
```suggestion
	const hasRedo = useSelect( ( select ) => select( 'core' ).hasRedo() );
```
I assume this should be using the `hasRedo` selector. 😄